### DISCUSSION TITLE
Software Volume Control Besides DDC Control

### DISCUSSION THREAD

**Discussion by owenzhao** [MonitorControl/MonitorControl#1231] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Monitor Control works great on brightness changing. However, for sound volume changing, that is another story. Sometimes sound control doesn't work for some monitors. For example:

* [Mute/Unmute is Reversed in My Monitor on Ventura](https://github.com/MonitorControl/MonitorControl/issues/1225)
* [Volume control is not working on my Dell u2720q display (brightness control does)](https://github.com/MonitorControl/MonitorControl/issues/1187)

In the latter link, I have learned that there are other apps that can control sound volume in software way. However, comparing to Monitor Control, these apps are lack of keyboard shortcuts and system OSD. One word, original system experience.

There is software control on brightness in Monitor Control. But for sound volume, there isn't. If Monitor Control could add this, that will be great. Although software control is not as direct operation as DDC. But I think original system experience is much more important.

### Describe the solution you'd like

nothing

### Describe alternatives you've considered

nothing

**Comment by waydabber** [comment]
This would be great but is rather difficult and requires a virtual audio device that acts as a filter to do this. This is definetly out of scope for this app. There are apps that can do this though (mostly paid - or for free there was Soundflower, I am not sure if it is functional anymore or if it has any free successor).
